spec/ORDERMANAGEMENT.mediawiki

```diff
@@ -0,0 +1,889 @@
+==Client Order Management==
+
+This section describes the steps required of the client to place an order, and
+the interactions between the client and server to execute an order once a match
+has been found.
+See the [[#atomic-settlement|atomic settlement]] section for a high-level
+overview of the settlement process.
+
+There are three available types of order.
+
+* [[#Limit_Order|Limit orders]] are used to buy or sell a specified amount of an asset at a rate no higher (buy) or lower (sell) than a specified price.
+* [[#Market_Order|Market orders]] are used to buy or sell a specified amount of an asset at the best price available.
+* [[#Cancel_Order|Cancel orders]] are used to remove standing limit orders from the order book.
+
+The order book holds only limit orders with time in force ''standing'' that have
+not completely filled or been cancelled. All other orders are only valid for one
+[[#Epochbased_Order_Matching|epoch match cycle]].
+
+===Session Authentication===
+
+Orders must be placed on an authenticated connection. Once a websocket
```

Comment:
Let's move this section to **communications protocols** and change this first sentence to read

> Many DEX messages must be sent on an authenticated connection.

Reply:
looks ok?